src/components/header/Header.jsx

```diff
@@ -32,6 +40,30 @@ export default class Header extends React.Component {
             userWidget = (
                 <UserMenu user={ userData }/>
             );
+
+            let callAssignments = this.props.assignments.getIn(['assignmentList', 'items']);
+
+            if (callAssignments.size > 0) {
+                let callUrl = '//call.' + process.env.ZETKIN_DOMAIN + '/';
+
+                callLink = (
+                    <Button href={  callUrl } labelMsg="header.call"
+                            className="Header-navLink linkCall"/>
+                );
+            }
+
+            let membership = this.props.orgs.getIn(['membershipList', 'items']);
```

Comment:
Probably should be called `memberships` to clarify that this is a list, and to be more consistent with the `callAssignments` variable which is essentially the same thing.